src/electron.ts

```diff
@@ -65,6 +65,16 @@ const createWindow = () => {
 		})
 	})
 
+	// Force-allow cross-origin requests, particularly for development mode,
+	// where the file is served from localhost
+	mainWindow.webContents.session.webRequest.onHeadersReceived((details, callback) => {
```

Comment:
What's this for 🤔

Reply:
This sets/overrides a CORS header to prevent issues fetching resources in development mode (in my case the bridge discovery was failing).